FirebaseMessaging/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# unreleased
+- [fixed] Fixed the issue that failed to download image if no extension but MIME type is set. (#6590)
+
```

Comment:
Fixed an issue that downloading an image failed when there's no extension in the file name but MIME type is set.

Reply:
Sounds good, updated language.